FreeSurfer/scripts/FreeSurferHiresPial.sh

```diff
@@ -75,13 +76,22 @@ cp --preserve=timestamps $SubjectDIR/$SubjectID/surf/rh.pial $SubjectDIR/$Subjec
 cp --preserve=timestamps $SubjectDIR/$SubjectID/surf/lh.thickness $SubjectDIR/$SubjectID/surf/lh.thickness.preT2.pass1
 cp --preserve=timestamps $SubjectDIR/$SubjectID/surf/rh.thickness $SubjectDIR/$SubjectID/surf/rh.thickness.preT2.pass1
 
-# Generate pial surfaces with T2 adjustment, still first pass.  
-# Same 4 files created as above, but have $outputSuffix1 ("postT2.pass1") appended through use of -output flag.
-# Use -T2 flag (rather than -T2dura), since -T2 is the flag used within recon-all script of FS 5.3 (but -T2 and -T2dura generate same results)
-#For mris_make_surface with correct arguments #Could go from 3 to 2 potentially...
-outputSuffix1=".postT2.pass1"
-mris_make_surfaces -nsigma_above 2 -nsigma_below 3 -aseg aseg.hires -filled filled.hires -wm wm.hires -mgz -sdir $SubjectDIR -orig white.deformed -nowhite -orig_white white.deformed -orig_pial pial -T2 "$mridir"/T2w_hires.norm -T1 T1w_hires.norm -output $outputSuffix1 $SubjectID lh
-mris_make_surfaces -nsigma_above 2 -nsigma_below 3 -aseg aseg.hires -filled filled.hires -wm wm.hires -mgz -sdir $SubjectDIR -orig white.deformed -nowhite -orig_white white.deformed -orig_pial pial -T2 "$mridir"/T2w_hires.norm -T1 T1w_hires.norm -output $outputSuffix1 $SubjectID rh
+if [ ! "${T2wImage}" = "NONE" ] ; then
+  # ceho "---> Generating pial first pass surfaces with T2 adjustment."
+  echo "---> Generating pial first pass surfaces with T2 adjustment."
+  # Generate pial surfaces with T2 adjustment, still first pass.
+  # Same 4 files created as above, but have $outputSuffix1 ("postT2.pass1") appended through use of -output flag.
+  # Use -T2 flag (rather than -T2dura), since -T2 is the flag used within recon-all script of FS 5.3 (but -T2 and -T2dura generate same results)
+  # For mris_make_surface with correct arguments #Could go from 3 to 2 potentially...
+  outputSuffix1=".postT2.pass1"
+  mris_make_surfaces -nsigma_above 2 -nsigma_below 3 -aseg aseg.hires -filled filled.hires -wm wm.hires -mgz -sdir $SubjectDIR -orig white.deformed -nowhite -orig_white white.deformed -orig_pial pial -T2 "$mridir"/T2w_hires.norm -T1 T1w_hires.norm -output $outputSuffix1 $SubjectID lh
+  mris_make_surfaces -nsigma_above 2 -nsigma_below 3 -aseg aseg.hires -filled filled.hires -wm wm.hires -mgz -sdir $SubjectDIR -orig white.deformed -nowhite -orig_white white.deformed -orig_pial pial -T2 "$mridir"/T2w_hires.norm -T1 T1w_hires.norm -output $outputSuffix1 $SubjectID rh
+else
+  #ceho "---> No T2w image, skipping generation of pial first pass surfaces with T2 adjustment."
```

Comment:
`ceho` doesn't even exist in the current state of the debug shlib.  I suggest not adding commented out commands.

Reply:
I have staged things for after the debug.shlib PR is accepted. I've now changed it to `verbose_red_echo`